### PR TITLE
Fix (some) Bengali issues

### PR DESCRIPTION
**Pull request by sftim**
Some fixes to make Bengali less broken

### CODE REVIEW

**Comment by sftim** [comment]
/language bn

@seokho-son FYI

**Comment by sftim** [comment]
Remaining build errors:

```
ERROR [bn] no Resource or Page matching "included/optional-kubectl-configs-pwsh.md*".
ERROR [bn] tabs include not found for path "docs/tasks/tools/included/optional-kubectl-configs-fish.md"
WARN  .File.UniqueID on zero object. Wrap it in if or with: {{ with .File }}{{ .UniqueID }}{{ end }}
WARN  .File.Path on zero object. Wrap it in if or with: {{ with .File }}{{ .Path }}{{ end }}
ERROR [bn] No resource bundle found. Create an empty headless bundle; create content/bn/_common-resources/index.md with "headless: true" in front matter. See https://gohugo.io/content-management/page-bundles/#headless-bundle
Built in 44493 ms
Error: error building site: logged 3 error(s)
```

File: hugo.toml [hunk 1]
```diff
@@ -305,6 +305,18 @@ i18nDir = "./data/i18n"
 [languages.en.params]
 description = "Production-Grade Container Orchestration"
 
+[languages.bn]
+title = "Kubernetes"
+weight = 15
```
Comment (dipesh-rawat) [inline hugo.toml:310]: Value for `weight` configuration may require fine-tuning to ensure proper sorting order for all languages.

**Comment by asem-hamid** [comment]
hey @sftim  do we need to localize them now

"included/optional-kubectl-configs-pwsh.md*".
"docs/tasks/tools/included/optional-kubectl-configs-fish.md"

**Comment by sftim** [comment]
> hey @sftim do we need to localize them now
> 
> `included/optional-kubectl-configs-pwsh.md*`. `docs/tasks/tools/included/optional-kubectl-configs-fish.md`

yes, I think so

**Comment by asem-hamid** [comment]
can you please give us the full detail what we need to localize and action we need to take .

**Comment by sftim** [comment]
> can you please give us the full detail what we need to localize and action we need to take .

That's quite a lot of work and I don't plan to provide that detail myself. However, I can aim to facilitate a discussion between the different contributors involved here.

**Comment by sftim** [comment]
I'm happy to cherry-pick commits onto my branch if they help. If so, let me know PR IDs or commit IDs to cherry pick.

**Comment by asem-hamid** [comment]
@sftim this 3 errors are fixed in this PR #45003 
image

**Comment by asem-hamid** [comment]
anything remains ?

**Review by mitul3737 — APPROVED** [review]

**Comment by sftim** [comment]
Approval:
- https://github.com/kubernetes/website/pull/44986#issuecomment-1925756768
- https://github.com/kubernetes/website/pull/44986#pullrequestreview-1861428308

/approve

**Pull request merged**